haec-user-service/src/modules/user/services/user.service.ts

```diff
@@ -46,14 +46,51 @@ export class UserService {
           email: input.email,
         },
       });
-
       if (await bcrypt.compare(input.password, user.hashedPass)) {
         return true;
       }
     } catch (error) {
       console.error(error);
     }
-
     return false;
   }
+
+// This will log the user ** IN **
+
+  public async loginUser(input: {
+    email: string;
+    password: string;
+  }): Promise<string> {
+    const verified = await this.verifyUser({ email: input.email, password: input.password})
+    if (verified == true){
+      // Generates random 21 character token with Crypto
+      const Crypto = require('crypto')
+      function randomString(size = 21) {
+        return Crypto
+          .randomBytes(size)
+          .toString('base64')
+          .slice(0, size)
+      }
+      const token = randomString()
```

Comment:
`randomString()` really should be a `private` function defined on this class and not defined within this function. If this doesn't make sense, happy to talk to you about it later this week.

Reply:
I think that I remember `dynamic/static` and `private/public` :
-`static` are predefined and `CAN'T take arguments`
-`dynamic` are actually called something else and `CAN take arguments` 
-`private` are only callable within the... `instance` in which they are defined?
-`public` are callable anywhere
Something like that. I'll look into it.